query-cloudtrail.sh

```diff
@@ -51,7 +62,7 @@ for bucket in $(aws s3 ls | grep federalist | awk '{print $3}'); do
   set -x
   aws cloudtrail lookup-events \
     --lookup-attributes AttributeKey=ResourceName,AttributeValue="$bucket" \
-    | jq -c "$JQ_TRANSFORM"
+    | jq "$JQ_QUERY"
```

Comment:
I would recommend `jq -r` if you plan on doing anything with the output so the output is rendered as raw JSON. Not a requirement, just a suggestion.

Reply:
same response as above